### PR TITLE
Bigtable: cosmetic cleanup of table admin

### PR DESCRIPTION
**Pull request by igorbernstein2**
* improve sample code
* improve javadoc
* silence warnings
* inline compose helpers
* copy instance admin's future unwrap code

### CODE REVIEW

**Comment by igorbernstein2** [comment]
@kevinsi4508 can you take a look when you have a chance?

File: google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java [hunk 1]
```diff
@@ -175,129 +193,194 @@ public Table createTable(CreateTableRequest request) {
...
    */
+  @SuppressWarnings("WeakerAccess")
   public ApiFuture<Table> modifyFamiliesAsync(ModifyColumnFamiliesRequest request) {
     return transformToTableResponse(
         this.stub.modifyColumnFamiliesCallable().futureCall(request.toProto(instanceName)));
   }
 
   /**
-   * Deletes the specified tableId
+   * Deletes the specified tableId.
```
Comment (kevinsi4508) [inline google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java]: nit: Deletes the table specified by tableId.

File: google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java [hunk 2]
```diff
@@ -175,129 +193,194 @@ public Table createTable(CreateTableRequest request) {
...
    * }</pre>
    */
+  @SuppressWarnings("WeakerAccess")
   public void deleteTable(String tableId) {
     awaitFuture(deleteTableAsync(tableId));
   }
 
   /**
-   * Deletes the specified tableId asynchronously
+   * Asynchronously deletes the specified tableId.
```
Comment (kevinsi4508) [inline google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java]: Asynchronously  deletes the table specified by tableId.

File: google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java [hunk 3]
```diff
@@ -175,129 +193,194 @@ public Table createTable(CreateTableRequest request) {
...
    * }</pre>
    */
+  @SuppressWarnings("WeakerAccess")
   public Table getTable(String tableId) {
     return awaitFuture(getTableAsync(tableId));
   }
 
   /**
-   * Gets the Table by tableId
+   * Asynchronously gets the table  metadata by tableId.
```
Comment (kevinsi4508) [inline google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java]: Remove an extra space between "table" and "metadata".

**Comment by igorbernstein2** [comment]
@pongad this is ready for you, please review when you have a moment

**Pull request merged**